Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -178,7 +173,7 @@ lemma irreducible_polynomial [IsDomain R] : Irreducible W.polynomial := by
 #align weierstrass_curve.irreducible_polynomial WeierstrassCurve.Affine.irreducible_polynomial
 
 -- Porting note (#10619): removed `@[simp]` to avoid a `simpNF` linter error
-lemma eval_polynomial (x y : R) : (W.polynomial.eval <| C y).eval x =
+lemma eval_polynomial (x y : R) : W.polynomial.evalEval x y =
```

Comment:
Good idea, changed to `polynomial_evalEval_*` and added deprecation.

Reply:
I won't touch the Jacobian and Projective files in this PR though, other than necessary fixes.

Reply:
Could you remind me why we prefer `polynomial_eval` over `eval_polynomial`? With dot notation we have `monic_polynomial` for `W.polynomial.Monic` but `polynomial_eval ...` for `W.polynomial.eval ...`.

Reply:
Good point. Originally I renamed `map_eval_polynomial` to `map_polynomial_eval` because the `map_*` lemmas (including auto-generated `map_a₁`) follows the inside-first, outside-last convention, so it's natural to rename `eval_polynomial` to `polynomial_eval` too. However, I've now removed `map_eval_polynomial`, so we don't have to stick to that convention when it comes to `polynomial` and it's better to follow the `monic`, `degree` lemmas etc. I'm going to rename to `evalEval_polynomial_*`.

Reply:
Thanks, and sorry for going back and forth!